Library/Homebrew/os/mac/xcode.rb

```diff
@@ -353,7 +353,7 @@ def detect_clang_version
       end
 
       def detect_version_from_clang_version
-        detect_clang_version&.sub(/^(\d+)00\./, "\\1.")
+        detect_clang_version&.sub(/^(\d+)0(\d)\./, "\\1.\\2.")
```

Comment:
I will likely split this into a separate PR. This affects CLT 12.5 support in general: the Clang version in the beta is currently `1205.0.22.6` which does not match the original regex.